### DISCUSSION TITLE
NextJS PostCSS + Electron

### DISCUSSION THREAD

**Discussion by assimovt** [vercel/next.js#10829] [Help]
Heya. I'm using NextJS + Electron and trying to follow https://nextjs.org/docs/advanced-features/customizing-postcss-config but as soon as I add purgeCSS in production mode, things start to break. 

Did anyone have luck using it with Electron? Specifically following this example: https://github.com/zeit/next.js/blob/canary/examples/with-tailwindcss/postcss.config.js

**Comment by horacioh** [comment]
have you seen [this example](https://github.com/vercel/next.js/tree/canary/examples/with-electron-typescript) ?

I have not tried the setup with tailwind, but I don't see why this will not work.
